user_documentation.md

```diff
@@ -34,6 +34,9 @@ The app offers four controls for precise drone movement:
 2. Aileron and Elevator: Adjust the drone's roll and pitch with the horizontal control wheel.
 3. Rudder: Control the drone's yaw with the vertical control wheel next to the horizontal one.
 
+## Advanced Flight Modes
```

Comment:
Those are some cool features to have in our application. We can merge this PR once we have a concrete plan to implement these features.